src/utils/helpers.js

```diff
@@ -111,7 +111,7 @@ export const prepareUrl = function (url, useembedVideos = true) {
     return `https://www.google.com/search?q=${url}`;
   }
 
-  url = /^http(s)?:\/\//.test(url) ? url : `http://${url}`;
+  url = /^http(s)?:\/\//.test(url) || /^file:\/\/\//.test(url) ? url : `http://${url}`;
```

Comment:
This change is unnecessary. It will return any file types as HTTP urls. You might want to revert this.

Reply:
It will not if you type a file in manually it will just display what you type if you prefix it with file:///